projects/storefrontlib/layout/a11y/keyboard-focus/autofocus/auto-focus.service.ts

```diff
@@ -14,9 +14,9 @@ export class AutoFocusService extends EscapeFocusService {
    * Returns the first focusable child element of the host element.
    */
   findFirstFocusable(
-    host: HTMLElement,
+    host: HTMLElement | null | undefined,
```

Comment:
I removed undefined from the type, and it still works. Why is it here?

Reply:
It is also called by `scrollToTarget` in `skip-link.service.ts`. `target` can be undefined

Reply:
makes sense! it ties to this here https://github.com/SAP/spartacus/pull/15658#discussion_r870762291 since we're saying to put | undefined instead